packages/tokens/src/color-aliases.json

```diff
@@ -183,10 +183,10 @@
 			"overlay": {
 				"primary": {
 					"$value": "{color.white.950}",
-					"dark": "{color.gray.900}"
+					"dark": "{color.gray.950}"
 				},
 				"secondary": {
-					"$value": "{color.gray.950}",
+					"$value": "{color.white.900}",
```

Comment:
hm, white 900 is semi transparent. I think maybe we mean to use `gray.0`? tho I'm not actually sure where we use secondary overlay tokens?

Reply:
Updated `Popover` to use secondary overlay since it should use its dark value.

Reply:
So had the value mirror how primary has white 950 and gray 950.